src/renderer/components/catalog/Catalog.tsx

```diff
@@ -95,11 +94,12 @@ const mapStateToProps = (state: RootState) => ({
         "publication/updateTags",
         "reader/setLastReadingLocation",
     ]),
+    location: state.router.location,
```

Comment:
Why use `mapStateToProps` instead of `withRouter()` HOC + `RouteComponentProps`? Is this a personal preference, or is there a technical reason?